pages/catalog/PKPCatalogHandler.php

```diff
@@ -52,18 +52,41 @@ public function authorize($request, &$args, $roleAssignments)
      */
     public function category($args, $request)
```

Comment:
Categories should be handled in a new `CategoriesHandler` and the `PKPCatalogHandler` can be removed. The `fullSize` and `thumbnail` methods can be moved to OMP's `ArticlesHandler`.

So I think we want specific handlers named after what they do:

```
ArticlesHandler
SectionsHandler
CategoriesHandler
```

Or the other option is to roll them all into `ArticlesHandler`. So it would be like:

```php
namespace PKP\pages\publication;

abstract class PublicationsHandler extends Handler
{
    // ...
}
```

```php
namespace APP\pages\articles;

use PKP\pages\publication\PublicationsHandler;

abstract class ArticlesHandler extends PublicationsHandler
{
    // ...
}
```

The up-side is that we can probably share code more easily if there's just one handler handling all of the `/<page>/` data. The down-side is that this handler will probably be pretty big! :laughing:  I'm ok with either approach.

Reply:
ok, I think we had a similar discussion about this before. Sounds logical, the name of the CatalogHandler is very misleading.